package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "foxyext",
+  "version": "1.2.13",
+  "description": "Web Extension for Foxy Agent",
+  "dependencies": {},
+  "devDependencies": {
+    "web-ext": "^2.2.2"
+  },
+  "scripts": {
+    "lint": "web-ext lint",
```

Comment:
Does this not work with eslint?  I'd like to use the same line that we have in the foxycli.

Reply:
We can create two linters:
- webext lint: mozilla (security) linting for extension
- eslint: code style

So it could be similar to: 
```json
"scripts": {
  "lint": "eslint .",
  "lint:ext": "web-ext lint",
  "test": "npm run lint && npm run lint:ext"
}
```

Reply:
Yes, that sounds good.